website/frontend/src/components/pages/Message/Message.tsx

```diff
@@ -63,7 +63,7 @@ export const Message: FC = () => {
         decodedPayload = createType.create(type, message.payload);
       }
 
-      setMessagePayload(JSON.stringify(decodedPayload));
+      setMessagePayload(JSON.stringify(decodedPayload, null, 4));
```

Comment:
```suggestion
      setMessagePayload(getPreformattedText(decodedPayload));
```
We have helper function

Reply:
thank you)